config/routes.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.routes.draw do
-  match ':puppetdb(/*puppetdb_url)', :to => 'puppetdb_foreman/puppetdb#index', :puppetdb => /d3\.v2|charts|v3|puppetdb/, :as => "puppetdb"
+  match ':puppetdb(/*puppetdb_url)', :to => 'puppetdb_foreman/puppetdb#index', :puppetdb => /d3\.v2|charts|v3|puppetdb|metrics|\/pdb\/meta\/v1\/version\/latest|pdb\/meta\/v1\/version/, :as => "puppetdb"
```

Comment:
Line is too long. [202/80]<br>Use the new Ruby 1.9 hash syntax.<br>Use `%r` around regular expression.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.